docs/learn/integration-patterns/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: 'Supported Integration Patterns'
+---
+
+# Supported Integration Patterns
+
+Integrating systems efficiently is critical for seamless data exchange and workflow automation. When working with the Rossum API, various integration patterns can be employed to suit different technical requirements and business needs. In this article, we explore five key integration methods—each with its own pros and cons—to help you determine the best approach for your system.
+
+## 1. Scheduled polling integration
+
+In this model, the target system schedules regular queries to the Rossum API to retrieve annotations that are ready for export.
+
+<b>Pros:</b>
+
+- Full control over query frequency, error handling, and resource management.
+
+<b>Cons:</b>
+
+- No real-time updates.
+- Requires tracking loaded annotations (Rossum’s “Confirmed” to “Exported” statuses help with this).
+- Hosting computational resources is necessary to run these scheduled queries.
+
+![Scheduled-Polling-Integration](img/Scheduled-Polling-Integration.png)
+
+## 2. Webhook-driven integration
+
+Here, the target system listens for real-time notifications via Rossum’s Webhook. When a notification is received, the system queries the Rossum API to retrieve the annotation data. This eliminates the need for a scheduled job and for tracking which annotations are ready, compared to approach #1 above. Notifications Webhook is provided by Rossum out of the box.
+
+<b>Pros:</b>
+
+- Real-time updates.
+- Ability to respond dynamically to status changes, customizing actions based on business logic.
+- Webhook setup is straightforward, requiring just a target URL and specified triggers.
+
+<b>Cons:</b>
+
+- Requires readiness to handle incoming webhook data.
+- Even after receiving the notification, an API query is still necessary to retrieve the full annotation (except for certain event types that include annotation data. Please see [documentation](https://elis.rossum.ai/api/docs/#webhook-events) and option #3).
+
+![Webhook-Driven-Integration](img/Webhook-Driven-Integration.png)
+
+## 3. Direct push integration using webhook with annotation data
+
+In this variation, Rossum’s Webhook delivers a direct push of data to the target system.
+It will be possible to do for the range of actions like: confirmation or export. For more see [documentation](https://elis.rossum.ai/api/docs/#webhook-events). But if you want to react only on status change the attached payload will have no content of the annotation.
+
+<b>Pros:</b>
+
+- Real-time updates.
+- Rossum provides the Webhook integration out of the box.
+
+<b>Cons:</b>
+
+- The system must be able to handle incoming requests.
+- Not all events deliver annotation data.
+- Limited customization of the Webhook logic, as it is offered "as-is."
+
+![Direct-Push-Integration-(Option-#2)](<img/Direct-Push-Integration-(Option-2).png>)
+
+## 4. Direct push integration using serverless function hosted in Rossum
```

Comment:
Why not use export pipelines? I would personally not recommend this unless absolutely necessary (which is very rare). 🤔